src/main/java/com/jay/shoppingmall/service/handler/FileHandler.java

```diff
@@ -19,112 +22,100 @@
+        }
+        if (!multipartFile.getContentType().equals("image/png") && !multipartFile.getContentType().equals("image/jpeg")) {
+            throw new FileException("JPEG, PNG 파일만 업로드해주세요");
+        }
 
-            //프로젝트 내에 저장하기 위해 절대 경로를 설정.
-            //File.separator는 OS마다 다른 path 구분자를 처리하기 위함.
-            String absolutePath = new File("").getAbsolutePath() + File.separator + File.separator;
+        //폴더명을 업로드 한 날짜로 변환 후 저장
+        LocalDateTime now = LocalDateTime.now();
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("yyyyMMdd");
+        String currentDate = now.format(dateTimeFormatter);
 
-            //파일 저장 세부 경로 설정.
-            String path = "images" + File.separator + currentDate;
-            File file = new File(path);
-            File thumbnailFile = new File(path + File.separator + "thumbnails");
+        //프로젝트 내에 저장하기 위해 절대 경로를 설정.
+        //File.separator는 OS마다 다른 path 구분자를 처리하기 위함.
+        String absolutePath = new File("").getAbsolutePath() + File.separator + File.separator;
 
-            //폴더가 존재하지 않으면 만듬
-            //없는 폴더에 파일을 넣으려고 하면 I/O 익셉션 발생
-            if (!file.exists()) {
-                file.mkdirs();
-            }
-            if (!thumbnailFile.exists())
-                thumbnailFile.mkdirs();
-
-            //모든 파일에 대해 처리
-                //파일의 확장자 추출
-                String originalFileExtension = "";
-                String contentType = multipartFile.getContentType();
-
-                //확장자가 없다면 처리 중지
-                    if (contentType.contains("image/jpeg"))
-                        originalFileExtension = ".jpg";
-                    else if (contentType.contains("image/png"))
-                        originalFileExtension = ".png";
-
-                String fileName = System.nanoTime() + "";
-                        //확장자를 붙여 저장하는 것은 보안에 취약.
-                        //+ originalFileExtension;
-
-                Image image = Image.builder()
-                        .originalFileName(multipartFile.getOriginalFilename())
-                        .fileSize(multipartFile.getSize())
-                        .filePath(path + File.separator + fileName)
-                        .fileExtension(originalFileExtension)
-                        .imageRelation(imageRelation)
-                        .foreignId(foreignId)
-                        .isMainImage(false)
-                        .build();
-
-                //업로드 한 파일 데이터를 지정한 파일에 저장.
-                file = new File(absolutePath + path + File.separator + fileName);
-                try {
-                    multipartFile.transferTo(file);
+        //파일 저장 세부 경로 설정.
+        String path = "images" + File.separator + currentDate;
+        File file = new File(path);
+        File thumbnailFile = new File(path + File.separator + "thumbnails");
+
+        //폴더가 존재하지 않으면 만듬
+        //없는 폴더에 파일을 넣으려고 하면 I/O 익셉션 발생
+        if (!file.exists()) {
+            file.mkdirs();
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Confirm that `mkdirs` was successful. Handle any failures that happened. If you fail silently, then you might encounter errors that are harder to debug. See the following diffs on GitHub that make this code change: [diff1](https://github.com/qunarcorp/qmq/commit/12f1d9d6a57a02bc2415cb2af9f4388a77b12c02#diff-6b67be6e17f24fdeddb7c725ac8ec5600d6e63be0bdca0963a83d530e6d55e35L36), [diff2](https://github.com/apache/jmeter/commit/1ec61206779b3f84df18516682b4791bf015178f#diff-cd74304763c097fbc085ac2bc952b46ab3843ac825cacacad89e8064296e955aL330)

Learn more: [File.mkdirs in the Java Platform, Standard Edition 8 API Specification](https://docs.oracle.com/javase/8/docs/api/java/io/File.html#mkdirs--)